code/modules/mob/living/carbon/xenomorph/castes/warlock/abilities_warlock.dm

```diff
@@ -403,9 +403,9 @@
 				carbon_victim.apply_damage(PSY_CRUSH_DAMAGE * 1.5, STAMINA, blocked = BOMB)
 				carbon_victim.adjust_stagger(5 SECONDS)
 				carbon_victim.add_slowdown(6)
-			else if(isvehicle(victim))
+			else if(isvehicle(victim) || ishitbox(victim))
 				var/obj/vehicle/veh_victim = victim
```

Comment:
this can't be typecast as a vehicle anymore

Reply:
So do you want it to use isarmored?

Reply:
Alright, should be fixed now